lnrpc/lightning.proto

```diff
@@ -3711,6 +3718,22 @@ message ListPermissionsResponse {
     map<string, MacaroonPermissionList> method_permissions = 1;
 }
 
+message NodeAnnouncementUpdateRequest {
+    // Features holds the set of features the node supports.
+    map<uint32, Feature> features = 1;
+
+    // Color is the node's color in hex code format.
+    string color = 2;
+
+    // Alias is the alias or nick name of the node.
+    string alias = 3;
+
+    // Addresses is a slice of all the node's known addresses.
+    repeated NodeAddress addresses = 4;
```

Comment:
Similar problem that the one with `features`. Should we simply substitute the current node addresses with the ones that are provided in the request or we need to keep the ones in the conf file for example.
Also, should we use the default port for those addresses that are only the IP, or we should not support not providing it

Reply:
I think the same "action" scheme I outlined above could work here, so turning it into a map of actions.